classes/system/elasticsearch/server/cluster.yml

```diff
@@ -48,31 +46,3 @@ parameters:
       snapshot:
         repo:
           path: /var/lib/elasticsearch/repo
```

Comment:
I'd add the snapshot:path to ${_param:elasticsearch_snapshot_path} since you splitted out